### PR TITLE
Prepare release v0.2.2

### PR DESCRIPTION
**Pull request by morrisallison**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.2.2
+
+* Updated several dependencies [#10][]
```
Comment (stefanor) [inline CHANGELOG.md:5]: Not the most useful changelog message.

Makes me wonder if the dependencies are too tight, if we have to keep doing that.

In general, I think linking to PRs from changelogs is a waste of time. The point of the changelog isn't to document every change, but to document the impact of changes on clients. If the changelog is suitably descriptive, you don't need to go read the PR (usually you don't need all the gory details). And if you do, it's easy enough to find by traversing git history.

Comment (morrisallison) [inline CHANGELOG.md:5]: I agree on the PR linking. It was requested of me, so I've kept at it.

As for the dependencies, it's just a quick and simple chore to me. They change often, they're already outdated. I'll just close this one.

Comment (stefanor) [inline CHANGELOG.md:5]: If the person who requested it wants to chat, I'd love to. I don't like giving contradictory code review, that just sucks for the PR submitter.

We should probably document examples. https://github.com/yola/wiki/wiki/Releasing-packages makes no mention of this particular detail. Neither does http://keepachangelog.com/

**Pull request closed without merging**